### ISSUE TITLE
cifar10 code，loss remains unchanged

### ISSUE THREAD

**Issue by zhangyu199802** [yjwu17/BP-for-SpikingNN#6]
about the pytorch version for implementing cifar10 tasks ，sorry i don't know where the bug is，loss remains unchanged，low accuracy

**Comment by Cristina1213** [comment]
hello, did you succeed?

**Comment by zhangyu199802** [comment]
hello, I gave up this.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yjwu17/STBP-for-training-SpikingNN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月18日(星期一) 晚上10:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yjwu17/STBP-for-training-SpikingNN] cifar10 code，loss remains unchanged (Issue #6)





 
hello, did you succeed?

**Comment by crashkk** [comment]
I also found the same problem,but have solved it by setting firing threshold much lower(around 0.25).Probably this issue came from  neurons' early death.So I am confused that whether 0.75 the best  hyperparameter selection?

**Comment by QianpengLi577** [comment]
@crashkk Hi, do you still remember the final acc for Cifar10？

**Comment by crashkk** [comment]
@QianpengLi577 hi, 70%-80% roughly,but with severe overfitting

**Comment by QianpengLi577** [comment]
> @QianpengLi577你好，大约70%-80%，但有严重的过度拟合

Thanks for your reply

**Comment by QianpengLi577** [comment]
> @QianpengLi577 hi, 70%-80% roughly,but with severe overfitting

Thanks for your reply
